tasks/install_targz.yml

```diff
@@ -0,0 +1,54 @@
+---
+#   Create folders
+    - name: Create userspace folders
+      file:
+        path: "{{ item.path }}"
+        state: directory
+        mode: "{{ item.mode | default('0755') }}"
+      loop:
+        - { path: "{{ instance_dist_path }}" }
+        - { path: "{{ app_work_dir }}" }
+        - { path: "{{ instance_conf_path }}" }
+        - { path: "{{ instance_run_path }}", mode: 777}
+        - { path: "{{ instance_systemd_path }}" }
```

Comment:
`instance_dist_path` -> `apps_dir`
`app_work_dir ` -> `data-dir` (`data-dir` == `/var/lib/tarantool`, workdir of instance -> `<data-dir>/<app-name>.<instance-name>`)
`instance_conf_path` -> `conf-dir` (`conf-dir` == `/etc/tarantool/conf.d`, conf of instance -> `<conf-dir>/<app-name>.<instance-name>.yml`)
... and so on

See https://github.com/tarantool/cartridge-cli#environment-variables